main/webapp/modules/core/langs/translation-fr.json

```diff
@@ -113,6 +113,7 @@
     "core-index-open/warning-data-file": "Vous devez indiquer un fichier ou une url pour télécharger les données.",
     "core-index-open/new-title": "Nouveau nom du projet :",
     "core-index-open/name": "Nom",
+    "core-index-open/no-results-message": "Sans résultats",
```

Comment:
I'd expect "Pas de résultats" instead. Generally, it's easier if code PRs only touch `translation-en.json` and leave the rest to translators.

Reply:
Both FR & ES translations removed